dev-python/python-slugify/python-slugify-1.2.6.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="A Python Slugify application that handles Unicode"
+HOMEPAGE="https://github.com/un33k/python-slugify https://pypi.org/project/python-slugify/"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-python/unidecode[${PYTHON_USEDEP}]"
+DEPEND="${REDEPEND}"
```

Comment:
`REDEPEND` typo :)

Besides this, if `dev-python/unidecode` is needed at build time, then (in case of Python modules) appropriate variable would be `BDEPEND` instead of `DEPEND`.

Reply:
Thanks, fixed!